code/game/machinery/jukebox.dm

```diff
@@ -166,7 +270,7 @@
 
 /obj/machinery/media/jukebox/proc/explode()
 	walk_to(src,0)
-	src.visible_message(SPAN_DANGER("\the [src] blows apart!"), 1)
+	src.visible_message("<span class='danger'>\the [src] blows apart!</span>", 1)
```

Comment:
Lost the `span`

Reply:
Where?

Reply:
Should have clarified, lost the span macro.